src/components/ui/Dialog.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div class="text-center">
+    <v-dialog
+      origin="center top"
+      v-model="getIsOpen"
+      @click:outside="toggleDialog({ title: undefined, components: undefined })"
+    >
+      <v-card :light="!$vuetify.theme.dark" :dark="$vuetify.theme.dark">
+        <v-btn
+          icon
+          absolute
+          right
+          @click="toggleDialog({ title: undefined, components: undefined })"
+        >
+          <v-icon>mdi-close</v-icon>
```

Comment:
the icon of the dialogue is slightly missplaced in Firefox. Maybe create an issue for this?

Reply:
Could you take a screenshot of it for me, I couldn't find out what you mean

Reply:
There you go:

![image](https://user-images.githubusercontent.com/598854/90311286-9b2f1b00-def9-11ea-91b3-f4b9a540cfcd.png)

Reply:
Created the bug #79